cfgov/unprocessed/js/routes/on-demand/simple-chart/chart-hooks.js

```diff
@@ -132,6 +132,32 @@ const hooks = {
     return newData.sort( ( a, b ) => new Date( a.date ) - new Date( b.date ) );
   },
 
+  cct_age( data ) {
+    const raw = {};
+    const adjusted = {};
+    data.forEach( v => {
+      let currRaw, currAdj;
+      if ( raw[v.date] ) {
+        currRaw = raw[v.date];
+        currAdj = adjusted[v.date];
+      } else {
+        currRaw = raw[v.date] = { date: v.date, adjusted: 'Unadjusted' };
+        currAdj = adjusted[v.date] = { date: v.date, adjusted: 'Seasonally adjusted' };
+      }
+      currRaw[v.age_group] = v.vol_unadj;
```

Comment:
If the data's shape *doesn't* end up changing we can probably combine all these transforms into a reusable one by doing something like `currRaw[ Object.keys(currRaw)[4] ] = v.vol_unadj;`

Reply:
Yes, agreed. As of right now, the entire data shape is supposed to change soon and will be more consistent. If that ends up not happening, I will re-write all these hooks to make it easier.